blogposts/2021/dev-tool-time-adam-gordon-bell.md

```diff
@@ -0,0 +1,103 @@
+
+![Adam’s 45lb weight heatsink](https://sourcegraphstatic.com/blog/dev-tool-time/adam-gordon-bell/heatsink.jpeg)
+
+As the host of the popular tech podcast [CoRecursive](https://corecursive.com/), Adam uses a [MixPre-6 audio interface](https://www.sounddevices.com/product/mixpre-6-ii/) and the popular [Electro-Voice RE20 microphone](https://products.electrovoice.com/na/en/re20), and has acoustic paneling in his office.
+
+## The cost of customization
+
+> “If you’re out there and you have a crazy custom keyboard and you’re using the permissive hold feature, just remember that I built that and then gave up on the whole thing.”
+
+Missing from Adam’s current setup is the custom-built and programmed split keyboard he used several years ago. In his focus on configuration, he had even contributed code to the keyboard’s software, code that people are still using today. After his keyboard broke, though, Adam discovered the unpleasantness of no longer being able to comfortably and easily use a standard keyboard, and so switched to using a regular Apple keyboard to avoid the same pain in the future.
+
+![Adam’s 2016 desk setup with custom keyboard](https://sourcegraphstatic.com/blog/dev-tool-time/adam-gordon-bell/2016-computer-ergodox.png)
+
+<figure>
+    <figcaption style="text-align: center; font-style: italic;">Adam’s custom keyboard and struggle</figcaption>
+    <div class="container my-4 video-embed embed-responsive embed-responsive-16by9">
+        <video controls src="https://sourcegraphstatic.com/blog/dev-tool-time/adam-gordon-bell/custom-keyboard-struggle.mp4"></video>
+    </div>
+</figure>
+
+## Writing as a developer
+
+> “When I started doing writing ... I want this kind of experience I was used to as a developer, where you have linting, compilers, and a build pipeline that tells you whether things work or not.”
+
+Adam’s main writing environment isn’t a word processor, or even a speciality Markdown app. It’s [VS Code](https://code.visualstudio.com/), configured with a few plugins and with easy access to a terminal to use command line utilities. Two tools are at the core of his workflow for writing Markdown, the markup language he regularly uses. [`markdownlint`](https://github.com/DavidAnson/markdownlint) analyzes Markdown files according to configurable rules to ensure consistent and well-formed Markdown. It also can automatically fix some common errors. [Vale](https://github.com/errata-ai/vale) provides grammar checking according to custom rules. While Adam uses some of his own rules, the Vale community, which includes large corporations, sources a large number of rules, including those to detect “corporate speak!”
+
+<figure>
+    <figcaption style="text-align: center; font-style: italic;">See Vale in action detecting "corporate speak"</figcaption>
```

Comment:
```suggestion
    <figcaption style="text-align: center; font-style: italic;">See Vale in action detecting “corporate speak”</figcaption>
```
I've changed the "dumb" quotation marks and apostrophes to curly throughout, missed this one!
https://handbook.sourcegraph.com/communication/content_guidelines/style_and_mechanics#apostrophes